app/css/components/service-list.css

```diff
@@ -4,11 +4,7 @@
 
 .service-list__item {
   display: flex;
-  padding: 0;
   position: relative;
-}
-
-.service-list__item[data-icon] {
```

Comment:
We have a default icon for unknown services which is a question mark. So yes, all services have an icon.